doc/content/reference/configuration/identity-server.md

```diff
@@ -150,6 +152,10 @@ There are several options to customize the requirements for user passwords.
 - `is.user-registration.password-requirements.reject-common`: Reject common passwords
 - `is.user-registration.password-requirements.reject-user-id`: Reject passwords that contain user ID
 
+There is also an option to disable login with credentials:
```

Comment:
Maybe clarify that this then only allows login with the configured external (OpenID Connect) providers.